astroquery/nasa_exoplanet_archive/setup_package.py

```diff
@@ -1,12 +1,6 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import
 
-import os
-
 
 def get_package_data():
-
-    paths_core = [os.path.join('data', '*.json'),
-                  os.path.join('tests', 'data', '*.csv')]
-
-    return {'astroquery.nasa_exoplanet_archive': paths_core}
+    return {}
```

Comment:
this file is not needed any more I think

Reply:
Removed!